pyccl/background.py

```diff
@@ -271,35 +272,65 @@ def rho_x(cosmo, a, species, is_comoving=False):
         species_types[species], int(is_comoving))
 
 
-def Sig_MG(cosmo, a):
-    """Redshift-dependent modification to Poisson equation for massless
-    particles under modified gravity.
+def mu_MG(cosmo, a, k=None):
+    """Redshift-dependent modification to Poisson equation under modified
+    gravity.
 
     Args:
-        cosmo (:class:`~pyccl.core.Cosmology`): a Cosmology object.
+        cosmo (:class:`~pyccl.core.Cosmology`): Cosmological parameters.
         a (float or array_like): Scale factor(s), normalized to 1 today.
+        k (float or array_like): Wavenumber for scale
 
     Returns:
-        float or array_like: Modification to Poisson equation under \
-            modified gravity at scale factor a. \
-            Sig_MG is assumed to be proportional to Omega_Lambda(z), \
+        float or array_like: Modification to Poisson equation \
+            under modified gravity at a scale factor. \
```

Comment:
Small note, I believe the original wording of this is intended to mean "at scale factor given by a", which I think gives more information than the new wording?

Reply:
I see now the meaning -- fixed!